rdr_service/dao/workbench_dao.py

```diff
@@ -518,8 +515,6 @@ def get_workspaces_with_user_detail(self, **kwargs):
             query = (
                 session.query(WorkbenchWorkspaceApproved, WorkbenchResearcher, WorkbenchWorkspaceUser.role,
                               snapshot_subquery.c.snapshot_id)
-                    .options(joinedload(WorkbenchWorkspaceApproved.workbenchWorkspaceUser),
-                             joinedload(WorkbenchResearcher.workbenchInstitutionalAffiliations))
```

Comment:
Maybe I'm misunderstanding something, but it seems like at least this one is still needed. Down at line 552 (or 556 before the changes) the access to the researcher's `workbenchInstitutionalAffiliations` would cause another query to lazy-load them.